tools/ProgressOnderwijsUtilsBenchmarks/MicroOrmBench/Program.cs

```diff
@@ -16,11 +16,11 @@ static void RunCurrentBenchmarks([NotNull] Benchmarker benchmarker)
         {
             ParameterizedSqlExecutor.ConstructWithoutExecuting(benchmarker);
             ParameterizedSqlExecutor.RunQuery(benchmarker);
+            ParameterizedSqlExecutor.RunWideQuery(benchmarker);
             //ParameterizedSqlExecutor.RunTvpQuery(benchmarker);
             //DapperExecutor.RunQuery(benchmarker);
-            //ParameterizedSqlExecutor.RunWideQuery(benchmarker);
-            //HandrolledAdoNetExecutor.RunQuery(benchmarker);
-            //HandrolledAdoNetExecutor.RunWideQuery(benchmarker);
+            HandrolledAdoNetExecutor.RunQuery(benchmarker);

```

Comment:
Sould these be left as is?

Reply:
Mij maakt het niet uit.  Typisch doen we dev in bursts, dus MRU is geen gekke caching strategy, en dat is het laten staan een beetje ;-).